proxy/js/features.mjs

```diff
@@ -20,7 +20,9 @@ const requireUniqueEntries = array => {
   const count = Object.groupBy(array, it => it[0]);
   if (Object.values(count).some(it => it.length > 1)) {
     const offendingEntries = Object.entries(count).filter(it => it[1].length > 1).map(it => it[0]).join(', ');
-    throw new Error(`entries must be unique, offending entries: ${offendingEntries}`);
+    if (offendingEntries) {
```

Comment:
Is this change needed? The check above should already check if duplicates exist.

Reply:
it seems like there's a problem with the duplicate check

icons used for composition seem to have no ID, but I couldn't understand why.... [adding this line](https://github.com/k-yle/OpenRailwayMap-vector/blob/eabc5b03f3597d67109668d8b041298a0fa514ab/proxy/js/features.mjs#L21) is another way to bypass this problem